broker/JobScheduler.js

```diff
@@ -74,6 +75,8 @@ class JobScheduler {
     process.on('message', (msg) => this.handleMessage(msg));
     logger.info(`Starting Service Fabrik Batch Job worker: ${cluster.worker.id} - ${process.pid}  @${new Date()}`);
     require('./lib/jobs');
+    logger.info('Initializing event listeners..');
```

Comment:
Can you please remove this logger? or make it debug? in the long run we see too many unnecessary log statements in our app.